schutzbot/Jenkinsfile

```diff
@@ -13,6 +13,10 @@ pipeline {
         BUILD_CAUSE = detect_build_cause()
         GCP_BUCKET = "osbuild-composer-testing"
         GCP_REGION = "us-east4"
+        AZURE_TENANT_ID = "1710d22c-ccf0-4421-8ba7-0135cfaecb90"
+        AZURE_SUBSCRIPTION_ID = "8d026bb1-2a65-454d-a88f-c896db94c4f8"
```

Comment:
Are any of these IDs secret (not any more anyway)? Just wondering whether someone could use them to upload to our account...

Reply:
Without a client ID and secret, these are just random UUIDs. https://stackoverflow.com/questions/45661109/are-azure-subscription-id-aad-tenant-id-and-aad-app-client-id-considered-secre